dbt/adapters/spark/impl.py

```diff
@@ -171,7 +171,7 @@ def list_relations_without_caching(
         return relations
 
     def get_relation(self, database: str, schema: str, identifier: str) -> Optional[BaseRelation]:
-        if not self.Relation.include_policy.database:
+        if not self.Relation.get_default_include_policy().database:
```

Comment:
@ChenyuLInx does this still make sense? Just looping you in for python models impact sanity check

Reply:
I am not familiar with include_policy, but as long as the relations are still the same, I think it won't affect python models